platform/ios/Integration Tests/Snapshotter Tests/MGLMapSnapshotterTest.m

```diff
@@ -106,20 +108,29 @@ - (void)testDeallocatingSnapshotterDuringSnapshot🔒 {
 //  dispatch_queue_t backgroundQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
 
     __weak __typeof__(self) weakself = self;
+    __block __weak MGLMapSnapshotter *weakSnapshotter;
 
     dispatch_async(backgroundQueue, ^{
```

Comment:
Although this is called `backgroundQueue` it's created with `dispatch_get_main_queue()` - is that WIP?

Reply:
I’m unsure. This test and the others introduced in mapbox/mapbox-gl-native#12355 all refer to `backgroundQueue`s that are actually the main queue.

Reply:
The tests do pass if I make them actual background queues though – would you prefer if I made that change here or keep the tests as-is for consistency?

Reply:
Yep, I think that was my original intention - I don't remember why it got commented out - I suspect that was during a test that was broken 🙁